CHANGELOG.rst

```diff
@@ -146,7 +146,7 @@ October 28, 2019
 
 Features
 --------
-* DataStax Apollo Support (PYTHON-1074)
+* DataStax Apollo (Astra) Support (PYTHON-1074)
```

Comment:
I think you can simply remove "Apollo" even if the ticket was created with that word.